sp800-63a/sec4_ial.md

```diff
@@ -154,90 +159,94 @@ The CSP SHOULD perform identity proofing in-person. The CSP MAY perform remote i
+	- Enrollment codes sent by means other than physical mail SHALL be valid for a maximum of 10 minutes; those sent to a postal address of record SHALL be valid for a maximum of 7 days but MAY be made valid up to 21 days via an exception process to accommodate addresses outside the direct reach of the U.S. Postal Service.  
+	- If delivery of the enrollment code was sent to an address of record that is not physical mail, the CSP SHALL send notification of proofing to a different address of record than the destination of the enrollment code. For example, if the CSP sends an enrollment code to a mobile phone of record, a notification of proofing will be sent to the physical address in records or obtained from validated and verified evidence, such as a driver's license.
 
-#### 4.5.1.7. Biometric Collection
+#### 4.4.1.7. Biometric Collection
 
 The CSP MAY collect biometrics for the purposes of non-repudiation and re-proofing. See [Section 5.2.3](#biometric_use) of SP 800-63B for more detail on biometric collection.
 
-#### 4.5.1.8. Security Controls
+>**MG: How is a biometric used for re-proofing? Isn't it just an alternate authenticator?**
+
+#### 4.4.1.8. Security Controls
 
 The CSP SHOULD employ appropriately tailored security controls from the moderate baseline of security controls defined in [[SP 800-53]](#SP800-53) or equivalent industry standard and SHOULD ensure that the minimum assurance requirements associated with the *moderate* baseline are satisfied.
 
-### <a name="antecedent"></a>4.5.2. IAL2 Antecedent Proofing Requirements
+### <a name="antecedent"></a>4.4.2. IAL 2 Antecedent Proofing Requirements
+
+Antecedent in-person identity proofing MAY be used, provided the prior proofing transaction is determined to be comparable to the process defined in [Section 4.4.1.](#normal).  See also [The Federal Bridge Certification Authority (FBCA) Certificate Policy (CP)](#fbcacp), Section 3.2.3.1 Authentication of Human Subscribers for Medium Assurance _and_ [FBCA Supplementary Antecedent, In-Person Definition](#fbcasup) for more details.
 
-Antecedent in-person identity proofing MAY be used provided the prior proofing transaction is determined to be comparable to the process defined in [Section 4.5.1.](#normal).  See also [The Federal Bridge Certification Authority (FBCA) Certificate Policy (CP)](#fbcacp), Section 3.2.3.1 Authentication of Human Subscribers for Medium Assurance _and_ [FBCA Supplementary Antecedent, In-Person Definition](#fbcasup) for more details.
+>**MG: we use passive voice here. Who makes the determination that it is comparable?**
```

Comment:
I think we can get rid of this requirement if we do the derived stuff the way we discussed last week.  Thoughts?